plugins/generator-1.18.2/forge-1.18.2/armor.definition.yaml

```diff
@@ -1,28 +1,132 @@
 templates:
   - template: armor.java.ftl
     name: "@SRCROOT/@BASEPACKAGEPATH/item/@NAMEItem.java"
+
+# Helmet item model
   - template: json/item.json.ftl
     writer: json
-    condition: enableHelmet
-    deleteWhenConditionFalse: true
+    variables: "item=helmet"
+    condition:
+      - enableHelmet
+      - "helmetItemRenderType #= 0"

```

Comment:
Checking if the render type is 0 is redundant, the `hasNormalModel()`/`hasToolModel()` methods already check if the model type is `BUILTIN`